spec/rails_helper.rb

```diff
@@ -0,0 +1,34 @@
+require 'devise'
+require 'spec_helper'
+ENV['RAILS_ENV'] ||= 'test'
```

Comment:
purpose of this line?

Reply:
Bhai the spec_helper sets up the basic configuration for RSpec tests and ENV['RAILS_ENV']  sets the Rails environment to test so it only uses the test environment.